backend/src/controllers/items.js

```diff
@@ -0,0 +1,84 @@
+const mongoose = require("mongoose")
+const Item = require("../models/Item")
+
+// Get All Items
+const getAllItems = async (req, res) => {
+    try {
+        const items = await Item.find()
+        res.status(200).json(items)
```

Comment:
Use `createReponse` to create responses that has a consistent shape. Also you can use `StatusCodes` module to improve the readability.

eg:
```js
createResponse(res, StatusCodes.OK, items)
```